src/models/tools/geometry/jxg-board.ts

```diff
@@ -68,7 +68,8 @@ function scaleBoundingBoxToElement(domElementID: string, changeProps: any) {
   if (!eltBounds || !(eltBounds.width > 0) || !(eltBounds.height > 0)) {
     eltBounds = { width: kGeometryDefaultWidth, height: kGeometryDefaultHeight } as ClientRect;
   }
-  const { boundingBox, unitX, unitY } = changeProps;
+  const { boundingBox } = changeProps;
+  const [unitX, unitY] = getAxisUnits(changeProps);
   const [xMin, , , yMin] = boundingBox || [kGeometryDefaultAxisMin, , , kGeometryDefaultAxisMin];
   const xMax = xMin + eltBounds.width / (unitX || kGeometryDefaultPixelsPerUnit);
   const yMax = yMin + eltBounds.height / (unitY || kGeometryDefaultPixelsPerUnit);
```

Comment:
Since `getAxisUnits()` handles the defaulting now, you can remove the `|| kGeometryDefaultPixelsPerUnit`.

Reply:
d'oh, yes, this was the whole point of the function 😬